.gitignore

```diff
@@ -4,3 +4,5 @@
 pkg
 Gemfile.lock
 coverage
+/.idea
```

Comment:
Not everybody uses Jetbrains editors.
While this does not hurt. its bad practice to clutter projects with local env settings.
best to put that in the home folder of the dev 

```
$ grep ign ~/.gitconfig
        excludesfile = ~/.gitignore_global
```

my two cents on the topic